src/features/grid-reference.feature

```diff
@@ -0,0 +1,36 @@
+@regression
+Feature: BNGP-785 Grid Reference
+
+    As as Site Provider
+    I need to enter the central grid reference of my site
+    So that the Operator is able to identify its location
+
+    Background:
+        Given I am on the "grid-reference" page
+
+    Scenario Outline: I can add a valid grid reference
+        Then I should be able to add a valid grid reference "<grid-reference>"
```

Comment:
Ditto with this, doesn't check the value has been added to the submission does it?